src/components/header/header.js

```diff
@@ -12,9 +12,7 @@ class HeaderComponent extends LitElement {
 
       <header>
 
-        <a href="https://projectevergreen.github.io/">
-          <img src=${logo} alt="Project Evergreen logo"/>
-        </a>
+        <a href="https://projectevergreen.github.io/" style="background-image: url(./${logo})">&nbsp;</a>
```

Comment:
curious if this is more / less semantic and / or accessible?  👓

Reply:
@thescientist13 
I added a `title` attribute to the link to replace the removed `alt` attribute. Not exactly sure what the fallout is though.